### ISSUE TITLE
Mention of DOM streams in Flutter docs for StreamSubscription.pause is confusing

### ISSUE THREAD

**Issue by Hixie** [dart-lang/sdk#27737]
The docs for `StreamSubscription.pause` say:

> Currently DOM streams silently drop events when the stream is paused. This is a bug and will be fixed.

I'm not sure what this means, but I presume it doesn't apply to Flutter. We should probably find a way to exclude this comment from dartdocs that are for environments where it doesn't apply.

**Comment by lrhn** [comment]
Agree. If anything, it should be a comment on DOM streams, not here.
Unless it has been fixed, the it should just go away.
